webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,21 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      var savedValue = this.value;
+      this.value = "";
+      this.baseScrollHeight = this.scrollHeight;
+      this.value = savedValue;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = this.getAttribute("data-min-rows") | 0, rows;
+      this.rows = minRows;
+      rows = Math.ceil((this.scrollHeight - this.baseScrollHeight) / 17);
```

Comment:
17 might be for font size(I maybe wrong). But I tested different values on my machine. It works fine with all the values that I used.